### ISSUE TITLE
bloc_library fails to build

### ISSUE THREAD

**Issue by ernestoyaquello** [brianegan/flutter_architecture_samples#163]
There is already a [similar issue](https://github.com/brianegan/flutter_architecture_samples/issues/136), but it was closed.

When I try to build the `bloc_library`, I get the following error:

```
Running "flutter pub get" in bloc_library...                    
Because test >=1.6.2 <1.6.4 depends on test_api 0.2.5 and every version of flutter_driver from sdk depends on test_api 0.2.11, test >=1.6.2 <1.6.4 is incompatible with flutter_driver from sdk.

So, because bloc_library depends on both flutter_driver any from sdk and test 1.6.2, version solving failed.
pub get failed (1; So, because bloc_library depends on both flutter_driver any from sdk and test 1.6.2, version solving failed.)
Process finished with exit code 1
```

I am using Dart version 2.7.0 and Android Studio.

Any ideas?

**Comment by evandrom** [comment]
I'm having this error too.

**Comment by hunght** [comment]
same here, someone can help with this, pls

**Comment by brianegan** [comment]
Hey all -- thanks for the reports! On break for the holidays, I'll patch this up next week when I'm back from vacation.

**Comment by fvvliet** [comment]
same same

**Comment by brianegan** [comment]
Should be all fixed up now! Please let me know if you run into trouble :)

**Comment by cyberrob** [comment]
Wow! Thanks for quick fix. But don't know which commit fix it..Sorry for not familiar configs.

**Comment by brianegan** [comment]
Hey, sorry about that -- unfortunately I had to wrap this fix in a much larger fix to update all samples to the latest version of Flutter. The issue basically came down to resolving version conflicts between the dependencies in the bloc_library package (test) and the latest version of Flutter. All I did was open up the dependency ranges for the package itself so the pub tool could version solve a compatible version!
